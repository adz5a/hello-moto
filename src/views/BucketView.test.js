import {
    folders
} from "./BucketView";

it("can render folders", () => {

    const prefixes = {
        "a": {
            "a/b": {
                "a/b/c": {}
            }
        }
    };

    expect(folders(prefixes)).toMatchSnapshot();

});
