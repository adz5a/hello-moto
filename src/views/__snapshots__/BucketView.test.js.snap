// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render folders 1`] = `
Array [
  <li>
    <span>
        a
    </span>
    <ul>
        <li>
            <span>
                a/b
            </span>
            <ul>
                <li>
                    <span>
                        a/b/c
                    </span>
                </li>
            </ul>
        </li>
    </ul>
</li>,
]
`;
